### ISSUE TITLE
Make Copyright Year Dynamic with using javascript

### ISSUE THREAD

**Issue by parammittal16** [Software-Incubator/IceCream#45]
Copyright year in the footer could be made dynamic using javaScript so no need to update manually every year

**Comment by parammittal16** [comment]
@bvishal8510 @harshs14 @Stark-Pramod What do you think ?

**Comment by harshs14** [comment]
@parammittal16, yes sir, making it dynamic will be suitable. Will surely work on this.
